projects/openapi-io/src/validation/validator.ts

```diff
@@ -68,7 +71,8 @@ export const processValidatorErrors = (
     if (
       pathsWithErrors.every(
         (addedError) => !addedError.instancePath.startsWith(error.instancePath)
-      )
```

Comment:
right now we reduce polymorphic branch errors down to the first case. This isn't correct, but it's reasonable to give users a single thing to do instead of N conflicting things to do. Either way they have to go back and decide what they are trying to achieve. 

This lets any of our customer validator rules log their results too